ui/raidboss/data/04-sb/ultimate/ultima_weapon_ultimate.js

```diff
@@ -32,6 +32,51 @@ export default {
     },
   ],
   triggers: [
+    {
+      id: 'UWU Aetherochemical Laser Middle',
+      netRegex: NetRegexes.ability({ source: 'The Ultima Weapon', id: '2B84', capture: false }),
+      netRegexDe: NetRegexes.ability({ source: 'Ultima-Waffe', id: '2B84', capture: false }),
+      netRegexFr: NetRegexes.ability({ source: 'Ultima Arma', id: '2B84', capture: false }),
+      netRegexJa: NetRegexes.ability({ source: 'アルテマウェポン', id: '2B84', capture: false }),
+      netRegexCn: NetRegexes.ability({ source: '究极神兵', id: '2B84', capture: false }),
+      netRegexKo: NetRegexes.ability({ source: '알테마 웨폰', id: '2B84', capture: false }),
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          cn: '中间激光',
```

Comment:
I think not to say "what it is", prefer to say "where you should go"?
Also, pls add en as well, here and everywhere.

Reply:
There are different condition for this ability, there isn't a certain place a player should goto.

And, I'm not sure how to translate it is suitable. Should it be `middle laser`, `right laser` or something else.....

Reply:
Yep, just capitalize it. `Middle Laser`, `Right Laser`, and so on.

Reply:
I think it's complicated because it's left/right/middle, but coming from Ultima, who is in the east (I think?).  And the laser itself is a trident.

I think I am struggling a little with understanding how somebody can react to this callout.

Reply:
The  location of ultima is at East North and will always be East North in different battle at this phase.

I'm not sure I understand this right. The laser isn't trident. It's rectangle. The trident your saw in the screenshot is casted by Titan , which is another ability.

Reply:
![-7s28Q5-1rovZdT1kSdw-g4 png thumb](https://user-images.githubusercontent.com/13553903/104420617-18e47900-55b5-11eb-8c59-5f86ca9c42c2.jpg)

Reply:
Oh you're correct, sorry! I watched some videos and reread a guide and it is definitely not a trident.

I'm still not sure what to call these.  There's kind of "north laser" "middle laser" ... "east laser"? I personally think calling them "right laser" and "left laser" are going to confuse people.

Reply:
I have no opinion about this. I just find they have different effect id and I think we could add triggers to notice players.

Reply:
What about `Diagonal Laser` ?
I don't know if it popular in en tho.😂

Reply:
Sorry for my poor English... I mean I have no ideas which is better, not meaning north South is Worth.

Reply:
Sorry for delay, I'll just say "North Laser", "Middle Laser", "East Laser" for English.